manta-js/examples/extension-example/src/App.tsx

```diff
@@ -55,7 +61,11 @@ export default function App() {
     setZkAddress(accounts[0].zkAddress);
     setInjected(injected);
 
-    const provider = new WsProvider(rpcUrl);
+    // const provider = new WsProvider(rpcUrl);
+    const provider = new HttpProvider(
+      rpcUrl[(Math.random() * rpcUrl.length) | 0],
```

Comment:
I believe you need to do `Math.floor(Math.random() % rpcUrl.length)` or you are accessing list using floats

Reply:
`(Math.random() * 3) | 0`

`| 0`

Reply:
Ah gotcha the `| 0` truncates the value to integer